nilearn/regions/tests/test_signal_extraction.py

```diff
@@ -338,8 +338,8 @@ def test_signal_extraction_with_maps_and_labels():
 
     # Check that NaNs in regions inside mask are replaced with zeros
     region1 = labels_data == 2
-    indices = [ind[:1] for ind in np.where(region1)]
-    get_data(fmri_img)[indices + [slice(None)]] = np.nan
+    indices = tuple(ind[:1] for ind in np.where(region1))
+    get_data(fmri_img)[indices] = np.nan
```

Comment:
Adding a `slice(None)` is just adding an extra `:`, which has no effect on the result. I just dropped it, but we could restore it by adding `(slice(None), )` instead.